data/udev/99-thrustmaster-pedal-perms.rules

```diff
@@ -0,0 +1,8 @@
+# Match kernel name of device, rather than ATTRS{idProduct} and ATTRS{idVendor}
```

Comment:
I'd rather put it in the thrustmaster udev file despite having "wheel" in its name. I don't feel like creating many more udev files.